### PR TITLE
Orders from users to group

### PR DESCRIPTION
**Pull request by enricostano**
### Screenshot :camera_flash: 
2016-11-27-164549_2560x1600_scrot
2016-11-27-164650_2560x1600_scrot

### Dependencies
Backend: https://github.com/coopdevs/katuma/pull/97

### TODO
 - [ ] Try out https://github.com/jkbrzt/rrule

### CODE REVIEW

File: src/containers/Group/Orders/index.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        <Helmet title={`Pedidos para ${group.name}`}/>
+
+        <h3>Pedidos</h3>
+        {this.renderOrders()}
+      </div>
+    );
+  }
+}
+
+const mapStateToProps = (_state) => {
```
Comment (enricostano) [inline src/containers/Group/Orders/index.js]:
```
[0] /home/ubuntu/katuma-web/src/containers/Group/Orders/index.js
[0]   57:26  error  "_state" is defined but never used  no-unused-vars
[0] 
[0] ✖ 1 problem (1 error, 0 warnings)
```

@andresgutgon @naoisegolden @edulan como quito eso, que quiere de mi?

Comment (edulan) [reply src/containers/Group/Orders/index.js]: quiere tu :moneybag:

Comment (edulan) [reply src/containers/Group/Orders/index.js]: http://eslint.org/docs/rules/no-unused-vars#varsignorepattern

Comment (edulan) [reply src/containers/Group/Orders/index.js]:
``` json
"no-unused-vars": ["error", { "varsIgnorePattern": "^_" }]
```

Comment (enricostano) [reply src/containers/Group/Orders/index.js]: Pero cual es la diferencia con este https://github.com/coopdevs/katuma-web/blob/develop/src/containers/Group/Producers/List.js#L149

tampoco alli se usa `_state`, verdad?

Comment (edulan) [reply src/containers/Group/Orders/index.js]: Tiene que ver con la opción "after-used" que viene por defecto:

> after-used - only the last argument must be used. This allows you, for instance, to have two named parameters to a function and as long as you use the second argument, ESLint will not warn you about the first. This is the default setting.

http://eslint.org/docs/rules/no-unused-vars#args

File: package.json [hunk 2]
```diff
@@ -21,7 +21,6 @@
     "start": "concurrent --kill-others \"yarn run start-prod\"",
     "start-prod": "node ./node_modules/better-npm-run start-prod",
     "build": "webpack --verbose --colors --display-error-details --config webpack/prod.config.js",
-    "postinstall": "webpack --display-error-details --config webpack/prod.config.js",
```
Comment (andresgutgon) [inline package.json:24]: Why?

File: src/containers/Group/Orders/CurrentOrder/OrderLines/Base.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+    return (
+      <div>
+        <List orderLines={orderLines} grandTotal={grandTotal} />
+      </div>
+    );
+  }
+}
+
+export default OrderLinesBase;
```
Comment (andresgutgon) [inline src/containers/Group/Orders/CurrentOrder/OrderLines/Base.js]: This component is not necessary. It's doing nothing you could use directly 'Lis' component

File: src/containers/Group/Orders/CurrentOrder/index.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    loadProducts: PropTypes.func.isRequired,
+    loadOrderLines: PropTypes.func.isRequired,
+    order: PropTypes.object.isRequired,
+    orderLines: PropTypes.array.isRequired,
+    grandTotal: PropTypes.number.isRequired,
+  }
+
+  constructor(props) {
+    super(props);
+  }
```
Comment (andresgutgon) [inline src/containers/Group/Orders/CurrentOrder/index.js]: :fire:

File: src/containers/Group/Orders/CurrentOrder/index.js [hunk 5]
```diff
@@ -0,0 +1,58 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import moment from 'moment';
+
+import { load } from 'redux/modules/products/';
+import { load as load2 } from 'redux/modules/order_lines';
```
Comment (andresgutgon) [inline src/containers/Group/Orders/CurrentOrder/index.js]: Better name please

File: src/containers/Group/Orders/Item/index.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
```
Comment (enricostano) [inline src/containers/Group/Orders/Item/index.js:2]: remove `connect`

File: src/containers/Group/Orders/List/index.js [hunk 7]
```diff
@@ -0,0 +1,35 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
```
Comment (enricostano) [inline src/containers/Group/Orders/List/index.js:3]: remove `connect`

File: src/containers/Group/Orders/CurrentOrder/AddProductsModal/index.js [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+    const { showModal, onCloseModal } = this.props;
+
+    return (
+      <Modal show={showModal} onHide={onCloseModal}>
+        <Modal.Header closeButton>
+          <Modal.Title>Elige tus productos</Modal.Title>
+        </Modal.Header>
+
+        <Modal.Body>
+          HOLA
```
Comment (sauloperez) [inline src/containers/Group/Orders/CurrentOrder/AddProductsModal/index.js]: :trollface:

Comment (edulan) [reply src/containers/Group/Orders/CurrentOrder/AddProductsModal/index.js]: ke ase

Comment (enricostano) [reply src/containers/Group/Orders/CurrentOrder/AddProductsModal/index.js]: que pasa? no se puede ya ni ser educados y saludar?

**Review by andresgutgon — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (enricostano) [reply package.json:24]: Because it's slow and we don't need it

File: src/containers/Group/Orders/CurrentOrder/OrderLines/Base.js (hunk 3 above)
Comment (enricostano) [reply src/containers/Group/Orders/CurrentOrder/OrderLines/Base.js]: Yep, I know. I was just doing some experiment.

**Review by andresgutgon — APPROVED** [review]

**Pull request merged**